components/mfk/mfk.py

```diff
@@ -480,14 +481,10 @@ def __init__(self, rc_json):
         self._json = rc_json
         if "version" in rc_json:
             self.version = float(rc_json["version"])
-            if self.version == 0.1:
-                self.components = list(map(Component, rc_json["components"]))
-            elif self.version == 0.2:
-                self.components = list(map(Component, rc_json["data"]["components"]))
-            else:
-                raise ValueError('unhandled version {0}'.format(self.version))
-        else:
+        if "data" in rc_json:
```

Comment:
is this the latest mfk.py?  I thought that the version info was maintained after all?  it should be kept